src/jsonstream.cpp

```diff
@@ -0,0 +1,201 @@
+//
+// Created by jpiotte on 10/23/19.
+//
+
+#include <simdjson/isadetection.h>
+#include <atomic>
+#include <simdjson/padded_string.h>
+#include "simdjson/jsonstream.h"
+
+using namespace simdjson;
+
+//Architecture find_best_supported_implementation();
+//int json_parse_implementation(ParsedJson &pj, bool realloc_if_needed);
+//
+//
+//// function pointer type for json_parse
+//using json_parse_functype = int(const char *buf, size_t len, size_t batch_size, ParsedJson &pj, bool realloc_if_needed);
+//
+//// Pointer that holds the json_parse implementation corresponding to the
+//// available SIMD instruction set
+//extern std::atomic<json_parse_functype *> json_parse_ptr;
+//
+
+
+
+
+
+
+JsonStream::JsonStream(const char *buf, size_t len, size_t batch_size) {
```

Comment:
It's a minor thing, but should we not use the initializer list here instead of initializing in the constructor body?